src/cartservice/Protos/Cart.proto

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package hipstershop;
+
+// -----------------Cart service-----------------
+
+service CartService {
+    rpc AddItem(AddItemRequest) returns (Empty) {}
+    rpc GetCart(GetCartRequest) returns (Cart) {}
+    rpc EmptyCart(EmptyCartRequest) returns (Empty) {}
+}
```

Comment:
I'm going to copy this comment from the other PR: it looks like these protos are a duplicate from the ones in `/pb`. Shouldn't we use those instead?

Reply:
With new .NET managed grpc stack, the protos are defined as part of the project (e.g. cartservice.csproj) and then it's being compiled and code is autogenerated. Referencing proto file from csproj such that proto is located outside of csproj folder or subfolders will not work. It will also make it problematic when building docker container.
Unfortunately, we need to keep the duplication of proto here.